priv/repo/migrations/20201103193047_create_upload.exs

```diff
@@ -0,0 +1,17 @@
+defmodule ChatApi.Repo.Migrations.CreateUpload do
+  use Ecto.Migration
+
+  def change do
+    create table(:upload, primary_key: false) do
```

Comment:
What about keeping the bounded-context name as `Uploads` and the Schema itself as `File`?